src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -468,11 +471,141 @@ private void lookupBranchReviewConceptChanges(BranchReview branchReview) {
 		start.setTime(start.getTime() + 1);
 
 		Set<Long> changedConcepts = createConceptChangeReportOnBranchForTimeRange(source.getPath(), start, source.getHead(), branchReview.isSourceParent());
+		changedConcepts.addAll(getContradictoryConceptIdentifiers(branchReview, source, target));
 		branchReview.setStatus(ReviewStatus.CURRENT);
 		branchReview.setChangedConcepts(changedConcepts);
 		branchReviewRepository.save(branchReview);
 	}
 
+	/*
+	 * Return Concept identifiers where one side of the merge has added a Description to a Concept, whilst the other
+	 * side has deleted the Concept entirely.
+	 * */
+	private Set<Long> getContradictoryConceptIdentifiers(BranchReview branchReview, Branch source, Branch target) {
```

Comment:
This method will find a deleted Concept on one side of the merge and a new Description (regardless of type) on the other side of the merge.

Reply:
This replaces the previous manual filter of `onlyDifferenceIsDescriptionSynonym`.

Reply:
Conflicts occur when the same Concept is touched on both branches. Therefore this method is called from both the Source's point of view and the Target's point of view.